src/main/java/com/softserve/actent/model/entity/User.java

```diff
@@ -81,31 +75,36 @@
 
     @NonNull
     @ManyToMany
+    @JsonBackReference
     @JoinTable(name = "user_categories",
             joinColumns = {@JoinColumn(name = "user_id")},
             inverseJoinColumns = {@JoinColumn(name = "interests_id")})
-    private Set<Category> interests = new HashSet<>();
+    private List<Category> interests;
+
 
     @NonNull
-    @NotBlank(message = StringConstants.EMPTY_USER_SEX)
-    @Column(nullable = false)
-    private String sex;
+    public enum sex {
+        MALE, FEMALE
+    }
 
     @NonNull
     @ManyToMany
-    private Set<Event> events = new HashSet<>();
+    @JsonBackReference
```

Comment:
Try to properly use mapper instead of jackson dependencies.

Reply:
done